hawkbit-dmf/hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/integration/AmqpMessageHandlerServiceIntegrationTest.java

```diff
@@ -882,6 +891,25 @@ public void downloadOnlyAssignmentAllowsActionStatusUpdatesWhenTargetReportsFini
         verifyAssignedDsAndInstalledDs(controllerId, distributionSet.getId(), distributionSet.getId());
     }
 
+    @Test
+    @Description("Messages that result into certain exceptions being raised should not be requeued. This message should forwarded to the deadletter queue")
```

Comment:
> result into certain exceptions

It is only `EntityAlreadyExistsException` so why not mentioning it?

Reply:
I added `IllegalArgumentException` to be tested as well